src/Types/UnbalancedTransaction.purs

```diff
@@ -0,0 +1,59 @@
+module Types.UnbalancedTransaction where
```

Comment:
Same here. (feel free to ignore `Types.Transaction` because it wasn't due to legacy, we'll sort it out later.)